meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source /usr/sbin/resin-vars
+
+SUPERVISOR_IMAGE_ID=$(docker inspect --format='{{.Id}}' $SUPERVISOR_IMAGE)
+SUPERVISOR_CONTAINER_IMAGE_ID=$(docker inspect --format='{{.Image}}' resin_supervisor)
+
+runSupervisor() {
+    docker rm --force resin_supervisor || true
```

Comment:
Also,if the removal failed and there is actually a container there (which would only happen with a docker bug or some other weird corruption), then the docker run command that follows will fail anyways (with the good old "name already taken" error)